app/controllers/bookings_controller.rb

```diff
@@ -30,5 +30,6 @@ def booking_params
 
   def set_workspace
     @workspace = Workspace.find(params[:workspace_id])
+    authorize @booking
   end
 end
```

Comment:
same question as for the workspace controller :)

Reply:
Patrick said the authorize @booking should be in set_booking 
and authorize @workspace should be in the set_workspace

Reply:
alright:)